### PR TITLE
Seismic Configuration (Verified Publisher) 

### PR DESCRIPTION
**Pull request by amit-chaudhari1**
This is an update to the configuration connector.

This will allow us to enable our clients to use Triggers (webhooks) in PA and will allow them to run flows on certain events in power automate.

### CODE REVIEW

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 1]
```diff
@@ -766,14 +921,14 @@
       "authorizationUrl": "https://auth.seismic.com/central/connect/authorize",
       "tokenUrl": "https://auth.seismic.com/central/connect/token",
       "scopes": {
-        "seismic.self.view seismic.self.manage seismic.user.view seismic.user.manage seismic.library.view seismic.library.manage seismic.configuration.view seismic.configuration.manage offline_access": "seismic.self.view seismic.self.manage seismic.user.view seismic.user.manage seismic.library.view seismic.library.manage seismic.configuration.view seismic.configuration.manage offline_access"
+        "seismic.webhook seismic.self.view seismic.self.manage seismic.user.view seismic.user.manage seismic.library.view seismic.library.manage seismic.configuration.view seismic.configuration.manage": "seismic.webhook seismic.self.view seismic.self.manage seismic.user.view seismic.user.manage seismic.library.view seismic.library.manage seismic.configuration.view seismic.configuration.manage"
```
Comment (gpsarkar) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json]: Do not delete offline_access

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 2]
```diff
@@ -766,14 +921,14 @@
...
       }
     }
   },
   "security": [
     {
       "oauth2-auth": [
-        "seismic.self.view seismic.self.manage seismic.user.view seismic.user.manage seismic.library.view seismic.library.manage seismic.configuration.view seismic.configuration.manage offline_access"
+        "seismic.webhook seismic.self.view seismic.self.manage seismic.user.view seismic.user.manage seismic.library.view seismic.library.manage seismic.configuration.view seismic.configuration.manage"
```
Comment (gpsarkar) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json]: Do not delete offline_access

File: certified-connectors/Seismic Configuration/apiProperties.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
           "identityProvider": "oauth2",
           "clientId": "PLACEHOLDER_CLIENTID",
           "scopes": [
-            "seismic.self.view seismic.self.manage seismic.user.view seismic.user.manage seismic.library.view seismic.library.manage seismic.configuration.view seismic.configuration.manage offline_access"
+            "seismic.webhook seismic.self.view seismic.self.manage seismic.user.view seismic.user.manage seismic.library.view seismic.library.manage seismic.configuration.view seismic.configuration.manage"
```
Comment (gpsarkar) [inline certified-connectors/Seismic Configuration/apiProperties.json]: Do not delete offline_access

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 4]
```diff
@@ -407,9 +406,160 @@
...
         }
       }
+    },
+    "/webhooks/v1/subscriptions": {
+      "x-ms-notification-content": {
+        "description": "Schema of the notification content"
+      },
+      "post": {
+        "summary": "Create a Webhook Subscription",
+        "description": "Create a subscriptions with",
```
Comment (gpsarkar) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json]: correct description

Comment (gpsarkar) [reply certified-connectors/Seismic Configuration/apiDefinition.swagger.json]:
```suggestion
        "description": "Create a webhook subscription for any Seismic webhook. See https://developer.seismic.com/seismicsoftware/docs/events for details.",
```

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 5]
```diff
@@ -407,9 +406,160 @@
           }
         }
       }
+    },
+    "/webhooks/v1/subscriptions": {
+      "x-ms-notification-content": {
+        "description": "Schema of the notification content"
+      },
+      "post": {
+        "summary": "Create a Webhook Subscription",
```
Comment (gpsarkar) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json]:
```suggestion
        "summary": "Create a webhook subscription",
```

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 6]
```diff
@@ -326,7 +325,7 @@
           {
             "in": "path",
             "name": "teamsiteId",
-            "description": "Teamsite Id",
+            "description": "TeamSite Id",
```
Comment (amit-chaudhari1) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json]:
```suggestion
            "description": "Teamsite Id",
```

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 7]
```diff
@@ -274,7 +273,7 @@
           {
             "in": "path",
             "name": "email",
-            "description": "Email",
+            "description": "Email Id",
```
Comment (amit-chaudhari1) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json]:
```suggestion
            "description": "Email",
```

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 8]
```diff
@@ -224,7 +223,7 @@
           "Privacy Management"
         ],
         "summary": "Email opt-outs - Add or update emails",
-        "description": "This action allows one or more email addresses to be added or updated in the tracking opt-out list.",
+        "description": "This action allows one or more email addresses to be added or updated in the tracking opt-out list. If the email address isn't already in the list, it will be added otherwise it will be updated.",
```
Comment (amit-chaudhari1) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json]:
```suggestion
        "description": "This action allows one or more email addresses to be added or updated in the tracking opt-out list.",
```

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 9]
```diff
@@ -163,7 +162,7 @@
           {
             "in": "body",
             "name": "body",
-            "description": "Body",
+            "description": "Body for Content Manager",
```
Comment (amit-chaudhari1) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json]:
```suggestion
            "description": "Body",
```

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 10]
```diff
@@ -151,7 +150,7 @@
           "Content Properties"
         ],
         "summary": "Content properties - Add a content property",
-        "description": "Add a content property and associate it to an existing teamsite.",
+        "description": "Add a content property and associate it to an existing teamsite. Provide a set of property values in a single step using the `domainOfValues` object.",
```
Comment (amit-chaudhari1) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json]:
```suggestion
        "description": "Add a content property and associate it to an existing teamsite.",
```

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 11]
```diff
@@ -83,7 +82,7 @@
           {
             "in": "body",
             "name": "body",
-            "description": "Body",
+            "description": "The body for Content Property",
```
Comment (amit-chaudhari1) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json]:
```suggestion
            "description": "Body",
```

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 12]
```diff
@@ -63,7 +62,7 @@
           "Content Properties"
         ],
         "summary": "Content properties - Add values to a content property",
-        "description": "Add one or more content property values to an existing content property.",
+        "description": "Add one or more content property values to an existing content property. Property values cannot be added to properties where an existing domain of values does not exist. Property values are case sensitive.\n#### Warning:\nAdding a domain of values to an existing property that does not have an domain of values can result in a loss of data. If this is desired please log into Seismic and create a domain of values to enable this functionality.",
```
Comment (amit-chaudhari1) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json]:
```suggestion
        "description": "Add one or more content property values to an existing content property.",
```

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 13]
```diff
@@ -25,13 +24,13 @@
...
   "paths": {
-    "/contentProperties/{contentPropertyId}/values": {
+    "/integration/v2/contentProperties/{contentPropertyId}/values": {
       "get": {
         "tags": [
           "Content Properties"
         ],
         "summary": "Content properties - Get values of a content property",
-        "description": "Provides the list of content property values for a given content property.",
+        "description": "Provides the list of content property values for a given content property. This will only return values for properties that have a defined list of values. Properties which have no defined domain of values and are filled in freeform by users are not provided here.",
```
Comment (amit-chaudhari1) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json]:
```suggestion
        "description": "Provides the list of content property values for a given content property.",
```

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 14]
```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "Seismic Configuration",
-    "description": "Seismic configuration connector provides set of actions required for adding or updating content properties, fetching teamsites and updating email opt-outs.",
+    "description": "Seismic configuration connector provides set of actions required for adding/updating content properties, fetching teamsites and updating email opt-outs.",
```
Comment (amit-chaudhari1) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json]:
```suggestion
    "description": "Seismic configuration connector provides set of actions required for adding or updating content properties, fetching teamsites and updating email opt-outs.",
```

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 15]
```diff
@@ -802,7 +957,7 @@
     },
     {
       "propertyName": "Categories",
-      "propertyValue": "Marketing;Sales and CRM"
+      "propertyValue": "Sales"
```
Comment (gpsarkar) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json]: "propertyValue": "Marketing;Sales and CRM"

Comment (amit-chaudhari1) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json]:
```suggestion
      "propertyValue": "Marketing;Sales and CRM"
```

**Comment by vmanoharas** [comment]
Hello @gpsarkar, kindly let me know the reason that why are you reviewing this connector here in GitHub?

@amit-chaudhari1

**Comment by gpsarkar** [comment]
Hello @vmanoharas, Me and @amit-chaudhari1 are from same organization. I reviewed to ensure the correctness of the changes that are committed in this pr. Hope that is fine.

**Comment by vmanoharas** [comment]
Hello @gpsarkar, 

Thank you very much for the clarification, could you please review files offline and submit the final copy here?

**Comment by gpsarkar** [comment]
@vmanoharas The files in this pr are final now. Should we close this and create a new pr instead?

**Comment by vmanoharas** [comment]
> @vmanoharas The files in this pr are final now. Should we close this and create a new pr instead?

Hello @gpsarkar, Yes, please close this PR and create a new PR.

**Pull request closed without merging**